pkgs/development/python-modules/sqlalchemy-views/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+  # required deps
+, setuptools
+, sqlalchemy
+  # test deps
+, pytestCheckHook
+, pytest-cov
+}:
+
+buildPythonPackage rec {
+  pname = "sqlalchemy-views";
+  version = "0.3.2";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    repo = pname;
+    owner = "jklukas";
+    rev = "5f959ed4a898068fd6073a452c5a13f4e19ea59e";
```

Use the tag instead.

```suggestion
    rev = "refs/tags/v${version}";
```

---

There is no tag, otherwise I would've used it!

---

https://github.com/jklukas/sqlalchemy-views/releases/tag/v0.3.2

---

Ah, looks like there is now!

---

Done.